integration-tests/src/test/java/org/zowe/apiml/integration/ha/ChaoticTest.java

```diff
@@ -0,0 +1,118 @@
+/*
+ * This program and the accompanying materials are made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Copyright Contributors to the Zowe Project.
+ */
+
+package org.zowe.apiml.integration.ha;
+
+import io.restassured.RestAssured;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+import org.zowe.apiml.util.DiscoveryRequests;
+import org.zowe.apiml.util.categories.ChaoticHATest;
+import org.zowe.apiml.util.config.ConfigReader;
+import org.zowe.apiml.util.config.DiscoveryServiceConfiguration;
+import org.zowe.apiml.util.config.GatewayServiceConfiguration;
+import org.zowe.apiml.util.http.HttpRequestUtils;
+
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.util.Collections;
+
+import static io.restassured.RestAssured.given;
+import static io.restassured.http.ContentType.JSON;
+import static org.apache.http.HttpStatus.SC_OK;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.core.Is.is;
+import static org.junit.jupiter.api.Assumptions.assumeTrue;
+import static org.zowe.apiml.util.SecurityUtils.getConfiguredSslConfig;
+
+/**
+ * Verify behaviour of the application under chaotic testing
+ */
+@ChaoticHATest
+public class ChaoticTest {
+
+    private GatewayServiceConfiguration gatewayServiceConfiguration;
+    private DiscoveryServiceConfiguration discoveryServiceConfiguration;
+
+    private final String DISCOVERABLE_GREET = "/api/v1/discoverableclient/greeting";
+    private final String SHUTDOWN = "/application/shutdown";
+    final int gatewayInstances = gatewayServiceConfiguration.getInstances();
```

Comment:
It's strange to use the configurations here when they are created in the setUp, so probably move it there as well?